apps/antalmanac/src/actions/AppStoreActions.ts

```diff
@@ -198,6 +198,15 @@ export const toggleTheme = (radioGroupEvent: React.ChangeEvent<HTMLInputElement>
     });
 };
 
+export const toggleShow24HourTime = (radioGroupEvent: React.ChangeEvent<HTMLInputElement>) => {
+    AppStore.toggleShow24HourTime(radioGroupEvent.target.value === 'true');
+    logAnalytics({
+        category: analyticsEnum.nav.title,
+        action: analyticsEnum.nav.actions.CHANGE_TIME_FORMAT,
+        label: radioGroupEvent.target.value,
+    });
+};
```

Comment:
Since this is specifically an event handler, I'd prefer if this was a memo-ized callback in a component. e.g.

```tsx
function TimeFormatMenu() {
  // handleChange and similar names help indicate that it's being used in the context of some form input
  const handleChange = useCallback((event: React.ChangeEvent<HTMLInputElement>) => {
    // ...
  }, [])
}
```